battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketServer.java

```diff
@@ -58,21 +58,21 @@ public void sendGameOverToOpponent() {
         server.sendGameOverToOpponent();
     }
 
-    private Queue<Point2D> receiveAllMovesWithoutSending() {
+    private List<Point2D> receiveAllMovesWithoutSending() {
         allMoves = "";
-        Queue<Point2D> cells = new LinkedList<>();
+        ArrayList<Point2D> points = new ArrayList<>();
```

Comment:
List<Point2D>
Also, you can create it closer to the place where you will use it.

Advice:
consider using immutable collections.
check out [Splitter](https://google.github.io/guava/releases/23.0/api/docs/com/google/common/base/Splitter.html)